home/app_manager.py

```diff
@@ -87,6 +103,20 @@ class AppManagerWidget(ipw.VBox):
                 </li>
             {% endfor %}
             </ul>
+
+            The compatibility of dependencies may break by other apps, you can uninstall and install this app again to fix it
+        </div>"""
+    )
+
+    DEPENDENCIES_INSTALL_INFO = Template(
+        """<div class="alert alert-info alert-dismissible">
+        <a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+            The following dependencies will be changed:
+            <ul>
+            {% for p in dependencies_to_install %}
+                <li> {{ p.installed if p.installed is not none else '[Not Installed]' }} --> {{ p.required }} </li>
+            {% endfor %}
+            </ul>
```

Comment:
Per comment above, I would add the following
```suggestion
            </ul>
            
            WARNING: Reinstalling previously installed dependencies may break already installed apps. If that happens, uninstall this app and reinstall the app that was broken.
```

Reply:
Okay, I like the warning, but the warning in the blue alert-info box may be strange :p I changed it, and let's show  demo next week see others' opinion.